### PR TITLE
[CloudPC]Update restore API request body

### PR DESCRIPTION
**Pull request by rongting**

### CODE REVIEW

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/manageddevice-bulkrestorecloudpc.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+## Request body
+In the request body, supply JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|managedDeviceIds|String collection|The IDs of the Cloud PC devices.|
+|restorePointDateTime|DateTimeOffset|The UTC time point for selected Cloud PCs to restore to. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: '2014-01-01T00:00:00Z'.|
+|timeRange|restoreTimeRange|The time range of restore point.|
```
[the remark this answers is not in the document]
Comment (RuiHou105) [reply api-reference/beta/api/manageddevice-bulkrestorecloudpc.md:54]: Added.

File: api-reference/beta/api/manageddevice-restorecloudpc.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST /me/managedDevices/{managedDeviceId}/restoreCloudPc
+POST /deviceManagement/managedDevices/{managedDeviceId}/restoreCloudPc
```
[the remark this answers is not in the document]
Comment (RuiHou105) [reply api-reference/beta/api/manageddevice-restorecloudpc.md:34]: Updated.

File: api-reference/beta/resources/cloudpcbulkremoteactionresult.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents cloud PC bulk remote action result.
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|failedDeviceIds|String collection|A list of all the managedDevice DeviceID's that completed the bulk action successfully.|
```
[the remark this answers is not in the document]
Comment (rongting) [reply api-reference/beta/resources/cloudpcbulkremoteactionresult.md:21]: should be managed device id.

File: api-reference/beta/api/manageddevice-bulkrestorecloudpc.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+}
+-->
+``` http
+POST https://graph.microsoft.com/beta/me/managedDevices/bulkRestoreCloudPc
+Content-Type: application/json
+Content-length: 123
+
+{
+  "managedDeviceIds": [
+    "String"
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/manageddevice-bulkrestorecloudpc.md:81]: This should be an example value of managedDeviceId, not the type of this property. Same as below.

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 5]
```diff
@@ -1,5 +1,55 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "d5711d39-7d15-4fac-9d6d-e81c2feae401",
+                    "ApiChange": "Member",
+                    "ChangedApiName": "restoring",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **restoring** member to the **cloudPcStatus** enumeration.",
```
[the remark this answers is not in the document]
Comment (rongting) [reply changelog/Microsoft.CloudManagedDesktop.json:10]: updated

**Review by AshleyYangSZ — APPROVED** [review]
Please  resolve conflicting.

**Pull request closed without merging**